### PR TITLE
repos: Add repository list to the README

### PR DESCRIPTION
**Pull request by gorozco1**
This change adds a list of distros and repositories to get Clear
Containers 3.0 packages.

These packages are generated in Open Build Service with the sources
listed in this repo.

Fixes: #40 

Signed-off-by: Geronimo Orozco <geronimo.orozco@intel.com>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -30,6 +30,31 @@ Included components
...
+
+* CentOS
+
+  * 7.x http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/CentOS_7/
+
+* RHEL
+
+  * 7.x http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/RHEL_7/
+
+* OpenSUSE
```
Comment (jodh-intel) [inline README.rst]: Is this missing a URL for SuSE?

File: README.rst [hunk 2]
```diff
@@ -30,6 +30,31 @@ Included components
...
+Repositories
+============
+
+* Ubuntu
+
+  * 16.04 http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/xUbuntu_16.04
+  * 16.10 http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/xUbuntu_16.10 
+  * 17.04 http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/xUbuntu_17.04
+
+* Fedora
```
Comment (jodh-intel) [inline README.rst]: Could you sort the list of distros alphabetically?

File: README.rst (hunk 1 above)
Comment (gorozco1) [reply README.rst]: yes is still missing we do not have packages built yet in the "non-staging" repo.

File: README.rst (hunk 2 above)
Comment (gorozco1) [reply README.rst]: yep

File: README.rst (hunk 1 above)
Comment (jodh-intel) [reply README.rst]: I'd probably remove this line then and raise an issue so we don't forget to add it once we have the URL.

**Comment by erick0z** [comment]
lgtm

**Review by jodh-intel — APPROVED** [review]

**Pull request merged**